Fix codecov test issues

When codecov workflow executes, it throws 3 errors saying it receives Bad Request from upstream.

```plain
=== RUN   TestFakeToUpCodeCoverage
Warning: 2023/10/09 11:23:24 [warning] TibiaCreaturesCreatureImpl called on invalid creature
    webserver_test.go:65: 
        	Error Trace:	/home/runner/work/tibiadata-api-go/tibiadata-api-go/src/webserver_test.go:65
        	Error:      	Not equal: 
        	            	expected: 200
        	            	actual  : 400
        	Test:       	TestFakeToUpCodeCoverage
    webserver_test.go:85: 
        	Error Trace:	/home/runner/work/tibiadata-api-go/tibiadata-api-go/src/webserver_test.go:85
        	Error:      	Not equal: 
        	            	expected: 200
        	            	actual  : 400
        	Test:       	TestFakeToUpCodeCoverage
    webserver_test.go:124: 
        	Error Trace:	/home/runner/work/tibiadata-api-go/tibiadata-api-go/src/webserver_test.go:124
        	Error:      	Not equal: 
        	            	expected: 200
        	            	actual  : 400
        	Test:       	TestFakeToUpCodeCoverage
--- FAIL: TestFakeToUpCodeCoverage (1.86s)
```

When adding print of **c** (`c, _ = gin.CreateTestContext(w)`) and **w** (`w = httptest.NewRecorder()`) it looks like this..

**tibiaCreaturesCreature**
```plain
c:
&{{0xc000084c40 174 400} <nil> 0xc00012c200 [{race demon}] [] -1  0xc000082d00 0xc0000b85e8 0xc0000b8600 {{0 0} 0 0 {{} 0} {{} 0}} map[]  [] map[] map[] 0}

w:
&{400 map[Content-Type:[application/json; charset=utf-8]] {"information":{"api":{"version":0,"release":"","commit":""},"timestamp":"2023-10-09T12:26:42Z","status":{"http_code":400,"error":20002,"message":"could not find creature"}}} false <nil> map[Content-Type:[application/json; charset=utf-8]] true}
```

**tibiaGuildsGuild**
```plain
c:
&{{0xc000084f80 171 400} <nil> 0xc0004c0f00 [{name pax}] [] -1  0xc0005a31e0 0xc00036ffe0 0xc0004ea000 {{0 0} 0 0 {{} 0} {{} 0}} map[]  [] map[] map[] 0}

w:
&{400 map[Content-Type:[application/json; charset=utf-8]] {"information":{"api":{"version":0,"release":"","commit":""},"timestamp":"2023-10-09T12:26:42Z","status":{"http_code":400,"error":20004,"message":"could not find guild"}}} false <nil> map[Content-Type:[application/json; charset=utf-8]] true}
```

**tibiaHighscores**
```plain
c:
&{{0xc000084b00 203 400} <nil> 0xc00012c400 [{world antica} {category experience} {vocation sorcerer} {page 4}] [] -1  0xc000082d00 0xc0004ea228 0xc0004ea240 {{0 0} 0 0 {{} 0} {{} 0}} map[]  [] map[] map[] 0}

w:
&{400 map[Content-Type:[application/json; charset=utf-8]] {"information":{"api":{"version":0,"release":"","commit":""},"timestamp":"2023-10-09T12:26:42Z","status":{"http_code":400,"error":11008,"message":"the provided page is larger than max amount of pages"}}} false <nil> map[Content-Type:[application/json; charset=utf-8]] true}
```

--- a/.github/workflows/codecov.yml
+++ b/.github/workflows/codecov.yml
@@ -16,11 +16,18 @@ jobs:
         with:
           go-version-file: go.mod
 
+      - name: Connect to Tailnet
+        uses: tailscale/github-action@v2
+        with:
+          args: --accept-dns=true
+          oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
+          oauth-secret: ${{ secrets.TS_OAUTH_SECRET }}
+          tags: tag:github-runner
+          version: 1.62.1
+
       - name: Runing tests for coverage
         run: |
           go test -race -coverprofile=coverage.out -covermode=atomic `go list ./... | grep -v vendor/` -v
-        env:
-          TIBIADATA_PROXY: ${{ secrets.TIBIADATA_PROXY }}
 
       - name: Uploading coverage to Codecov
         uses: codecov/codecov-action@v4

